src/RegionManager.js

```diff
@@ -0,0 +1,145 @@
+import _ from 'underscore';
+import Marionette from 'backbone.marionette';
+
+export default function() {
+
+   // Manage one or more related `Marionette.Region` objects.
+  Marionette.RegionManager = Marionette.Controller.extend({
```

Comment:
As ridiculous as this is.. can we change this to extend Object.  Same API, but then you can turn off Controller without turning off RegionManager